### PR TITLE
Document Raspberry Pi (armhf) .deb files

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**
Adds links to armhf .deb files

**Context: Fixes an issue? Related issues**
Related to: https://github.com/jamulussoftware/jamuluswebsite/issues/728

**Status of this Pull Request**
Ready for review

**What is missing until this pull request can be merged?**
Review by @gilgongo and @hoffie 

**Does this need translation?**

YES

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

File: _posts/2020-03-28-Server-Rpi.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ This guide shows you how to install Jamulus on a Raspberry Pi.
 
 ## Server Installation - Raspberry Pi
 
-Since there are no official armhf .deb files yet, you'll need to compile Jamulus for your Pi from source.
+Although there are official armhf .deb files ([GUI download]({{ site.download_root_link }}{{ site.download_file_names.deb-gui-armhf }}) [headless download]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }})) which are recommended to be used, you can also compile Jamulus for your Pi from source.
```
Comment (pljones) [inline _posts/2020-03-28-Server-Rpi.md]: How about something stronger?
```suggestion
We now recommend you use the official armhf .deb files ([GUI download]({{ site.download_root_link }}{{ site.download_file_names.deb-gui-armhf }}) [headless download]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }})).  However, you can still compile Jamulus for your Pi from source, if you wish.
```

Comment (ann0see) [reply _posts/2020-03-28-Server-Rpi.md]: Ok. Done. You're correct. It's better to hint at the official files.

File: wiki/en/Running-a-Server.md [hunk 2]
```diff
@@ -79,13 +79,14 @@ To run a Directory [read this guide](Custom-Directories)
 
 Most people run Jamulus as a "pure" Server on **hardware without audio** (e.g. on a 3rd party/cloud host) running Linux. The following steps assume you are familiar with the command line and Debian/Ubuntu or similar distribution using systemd. To run a server on Windows or on the desktop with a graphical user interface, [see this section](#servers-on-the-desktop).  
 
-If you want to run a Server on a Raspberry Pi, you will need to [compile from source](https://github.com/jamulussoftware/jamulus/blob/master/COMPILING.md). See also this [guide for Raspberry Pi](/kb/2020/03/28/Server-Rpi.html) maintained by Jamulus user fredsiva.
+If you want to run a Server on a Raspberry Pi (or a different armhf based device), you will need to download the armhf `.deb` files, not the default `amd64` ones you'd use on an Intel/AMD based machine.
```
Comment (pljones) [inline wiki/en/Running-a-Server.md]: To highlight that `armhf` and `amd64` are the equivalent terms:
```suggestion
If you want to run a Server on a Raspberry Pi (or a different armhf based device), you will need to download the `.deb` files for `armhf`, not the default `amd64` ones you'd use on an Intel/AMD based machine.
```

File: wiki/en/Running-a-Server.md [hunk 3]
```diff
@@ -79,13 +79,14 @@ To run a Directory [read this guide](Custom-Directories)
...
+
 
 ### Installation
 
-1. Download the [latest headless .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }})
+1. Download the [latest headless (amd64) .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }}) or, if you use a Raspberry Pi etc. download the [latest armhf .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }})
 1. Update apt to make sure you have a current list of standard packages: `sudo apt update`
-1. Install the Jamulus package: `sudo apt install ./{{ site.download_file_names.deb-headless }}`
+1. Install the Jamulus package: `sudo apt install ./{{ site.download_file_names.deb-headless }}` or for ARM: `sudo apt install ./{{ site.download_file_names.deb-headless-armhf }}`
```
Comment (pljones) [inline wiki/en/Running-a-Server.md]: This wording needs to be made more consistent -- either shorten the first point to just "ARM" or use "Pi etc" here, perhaps?

Comment (ann0see) [reply wiki/en/Running-a-Server.md]: of for armhf sounded strange.

```suggestion
1. Install the Jamulus package: `sudo apt install ./{{ site.download_file_names.deb-headless }}` or for Pi (armhf): `sudo apt install ./{{ site.download_file_names.deb-headless-armhf }}`
```

Comment (pljones) [reply wiki/en/Running-a-Server.md]: Maybe just "or for Pi (etc)", as `amd64` isn't mentioned for the first file?

Comment (ann0see) [reply wiki/en/Running-a-Server.md]: Done. See my latest commit

**Review by pljones — APPROVED** [review]

**Pull request merged**